### ISSUE TITLE
Ruby group capstone - Catalog of my things - Kanban Board

### ISSUE THREAD

**Issue by edgardsoza** [edgardsoza/Ruby-capstone-project#39]
We are three people in the team, Kola, Muhammed and Jose. 
Kola is Team Member 1
Muhammed is Team Member 2
Jose is Team Member 3
Link to project:
https://github.com/edgardsoza/Ruby-capstone-project/projects/1

**Comment by algerina** [comment]
Hi team, 

You've done an amazing job so far 👏 💯 👍
There are some issues that you still need to work on before starting your Capstone! 

## Highlights
  - The tasks are divided equally.

## Required Changes ♻️

- [ ] Most of your cards don't have a description while it is required, kindly add description to every card.
- [ ] It is required to give a number of points to every task and your group tasks cards don't have points. Kindly work on that issue then submit again so that your Kanban board can be reviewed. 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by bobb-Rob** [comment]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ Correct set of project kanban
✅ Neat Project cards


Cheers and Happy coding!👏👏👏
